index.html

```diff
@@ -65,7 +65,11 @@ <h2>View States</h2>
         transcend.doNotSell(event);
       };
       var resetDoNotSell = (event) => {
-        airgap.setConsent(event, { SaleOfInfo: true });
+        airgap.setConsent(
+          event,
+          { SaleOfInfo: true },
+          { confirmed: true, prompted: true },
```

Comment:
the function name is `resetdoNotSell` - does it make sense that we are setting confirmed + prompted to true if we're trying to reset the state

Reply:
good catch. I just removed the consent options for this call 👍

Reply:
note that this file is just stubs btw